doc/fingerprints.txt

```diff
@@ -22,3 +22,13 @@ pub   2048R/0E604491 2013-04-30
       Key fingerprint = 8657 ABB2 60F0 56B1 E519 0839 D9C4 D26D 0E60 4491
 uid                  Matt Caswell <matt@openssl.org>
 uid                  Matt Caswell <frodo@baggins.org>
+
+pub   rsa3072 2021-02-14
```

Comment:
This key is old.  It should instead be:
```
pub   rsa4096 2021-02-14 [SC]
      B7C1C14360F353A36862E4D5231C84CDDCC69C45
uid          Paul Dale <pauli@openssl.org>
```

Reply:
Could you please upload it to the keyserver?